toolchain/lower/file_context.cpp

```diff
@@ -60,8 +60,9 @@ auto FileContext::Run() -> std::unique_ptr<llvm::Module> {
 
   // Lower function declarations.
   functions_.resize_for_overwrite(sem_ir_->functions().size());
-  for (auto i : llvm::seq(sem_ir_->functions().size())) {
-    functions_[i] = BuildFunctionDecl(SemIR::FunctionId(i));
+  for (auto [i, pair] : llvm::enumerate(sem_ir_->functions().enumerate())) {
+    auto [id, _] = pair;
+    functions_[i] = BuildFunctionDecl(id);
```

Comment:
```suggestion
  for (auto [id, _] : sem_ir_->functions().enumerate()) {
    functions_[id.index] = BuildFunctionDecl(id);
```
We use `id.index` when accessing `functions_` elsewhere (eg, `FileContext::GetFunction`).